plover/gui_qt/main.py

```diff
@@ -43,6 +45,11 @@ def __init__(self, config, controller, use_qt_notifications):
         self._app = QApplication([sys.argv[0], '-name', 'plover'])
         self._app.setAttribute(Qt.AA_UseHighDpiPixmaps)
 
+        styleSheetPath = os.path.join(CONFIG_DIR, 'plover.qss')
+        if os.path.isfile(styleSheetPath):
+            with open(styleSheetPath, 'r') as styleSheetFile:
+                log.info('stylesheet: %s', styleSheetPath)
+                self._app.setStyleSheet(styleSheetFile.read())
```

Comment:
```suggestion
        stylesheet = os.path.join(CONFIG_DIR, 'plover.qss')
        if os.path.isfile(stylesheet):
            log.info('applying stylesheet: %s', stylesheet)
            with open(stylesheet, 'r') as fp:
                self._app.setStyleSheet(fp.read())
```

Reply:
Actually, since this is Python it's probably more idiomatic to use EAFP. Open it and catch FileNotFoundError.

Reply:
I like to keep the _Path_ part in the name: it is not the style sheet itself after all.